### ISSUE TITLE
Pressing any key with a button assigned makes an error sound.

### ISSUE THREAD

**Issue by TheAsdas** [OpenEmu/OpenEmu#1741]
Using OS X Yosemite 10.10 (14A389)
Pressing any key with a button assigned (Select, A, B, etc...) makes an error sound, like if i was pressing a random key on the desktop.

**Comment by Pretz** [comment]
I'm experiencing this too, 10.10.2 (14C106a), using a NES30 in B mode (bluetooth keyboard).
It seems to be triggered by going full screen; doesn't start happening until I zoom the game window to full screen.

**Comment by sotty115** [comment]
me too, it happens regardless of fullscreen or not

**Comment by digamejh** [comment]
It happens with on me too when going fullscreen--regardless of core used.  The issue remains until you quit the game, even if you exit fullscreen.
OS X 10.10.1

**Comment by TheAsdas** [comment]
It happened to me when I tried to connect a WiiMote and configure it (thing that doesn't works with this Mac version, apparently). I've never tried to connect one again since the last time i got this issue, and it never has happened again to me.

**Comment by amarmor** [comment]
Having this issue on Yosemite as well - any fixes?

**Comment by Crymsius** [comment]
Having the same issue. Only way to avoid it is... not going full screen =/

**Comment by cyco** [comment]
Alright, we need to find out what it is that you all have in common.
I see 4 people using yosemite, 3 of you say it only happens in full screen, 1 guy says it happens either fullscreen or windowed.

Do you use the popout or just the main library window?
Do you have controllers attached? If so, wired or bt? Which models?
Does it also happen if you don't have any other applications open (esp. chrome, or any apps that have anything to do with input, automation or remote controls)

**Comment by Crymsius** [comment]
So 
I'm on Yosemite.
Using the popout.
No controller.
It happen even if every other app is closed

The way it's triggered : it's okay on window mode => fullscreen triggers the bug and thus the error sound is played whatever key I press (assigned or not) => if I come back to windowed, the problem persists and the sound is still here.



Now, I've tested without the popout aaand it's working perfectly: going fullscreen, coming back windowed, no problem zero error sound.


tl;dr : issue only appears when switching to fullscreen from a popout.

**Comment by cyco** [comment]
Just tried with 1.0.4 release and it happened to me too. That means this is already fixed. Thanks for your help!

**Comment by sotty115** [comment]
I feel so stupid for asking this, but how do you not go into a popout menu?

**Comment by Crymsius** [comment]
In Preferences => Gameplay => uncheck "Always use popout gameplay window". => Done :)
